### ISSUE TITLE
radiksType=t on production SPA when fetching Users via fetchList

### ISSUE THREAD

**Issue by kkomaz** [hstove/radiks-server#1]
Awesome start to making developer life easier @hstove!  I'm sure this is going to be heavily adopted in the future!

I set up my heroku server + heroku RADIKS server and when testing locally on my front-end SPA it works fine when fetching for a Users list via `Users.fetchList()`

![screenshot of google chrome 2-8-19 5-23-03 pm](https://user-images.githubusercontent.com/8473884/52522022-fec5f180-2c4d-11e9-9b53-052214bcfb55.png)

However, when I deploy my SPA to production via netlify I noticed the query for users get shortened to a single character string `t`

![screenshot of google chrome 2-8-19 5-23-05 pm](https://user-images.githubusercontent.com/8473884/52522038-19986600-2c4e-11e9-9265-5c83910aadcc.png)

Not exactly sure what's going on but would love some insight on how to resolve this!

**Comment by hstove** [comment]
Great catch - I just discovered this as well when I deployed LetterMesh to production, and I haven't updated the documentation yet.

The issue is that your code is being minified. Radiks uses the model's class name when storing and querying data. In production, your code is minified, so the class name is just `t`.

You'll need to fix this by adding the static property `className` to your class. So it'll look like this:

~~~javascript
class Task extends Model {
  static className = 'Task';
}
~~~

However, in this case, you're using the `BlockstackUser` class, which is defined in Radiks. I just pushed a fix that adds the `className` to all internal classes. If you upgrade `radiks` to `0.1.4`, this should be resolved. This is an issue with `radiks`, not `radiks-server`, so only the client-side library needs to be updated.
